app/controllers/api/v1/books_controller.rb

```diff
@@ -5,6 +5,10 @@ class BooksController < ApplicationController
 
       before_action :authenticate_user!
 
+      rescue_from ActiveRecord::RecordNotFound do
+        render json: { error: "Book doesn't found" }
```

Comment:
you may  move this to a new controller called ApiController or into Application controller and use 
```
render json: { error: error.message }, status: :not_found
```